src/main/java/seedu/address/model/person/PersonDetailContainsKeywordPredicate.java

```diff
@@ -45,6 +48,12 @@ public boolean test(Person person) {
         } else if (PREFIX_TAG.equals(prefix)) {
             return person.getTags().stream()
                          .anyMatch(tag -> StringUtil.containsSubstringIgnoreCase(tag.tagName, keyword));
+        } else if (PREFIX_MATRIC_NUMBER.equals(prefix)) {
```

Comment:
Good addition! It really levels up our app. Good to see the foundation you built paying off.